### PR TITLE
Added MUI alerts to replace browser alerts ⚠️

### PR DESCRIPTION
**Pull request by warmachine028**
# Use MUI Alerts 📐
- I Pritam Kundu have worked for #51 

## Changes Made
- Resolved merge conflicts for #55 
- Used Material UI to add Snackbar Component
- Added Snackbar to repalce browser alerts
  - Login In
  - Sign Up
  - Image Upload Error
  - Sign In Error
  - Firebase Error
   
- Replaced all alerts and error messages and with Snackbar Notifications

## Screenshots:

- Logging In
logged in

- Logging Out
logout

- Error Messages
error

- Creating Post
uploaded

### CODE REVIEW

**Comment by narayan954** [comment]
Thanks Pritam, reviewing it right away

**Review by narayan954 — CHANGES_REQUESTED** [review]
image
 I believe merging caused this, some features were not properly merged. Can you please fix this

**Comment by narayan954** [comment]
there's already a new post button , we don't need a double

**Comment by warmachine028** [comment]
Should I remove the top purple `New Post Button`

**Comment by warmachine028** [comment]
Alright I identified the glitch. I accidentally deleted this chunk of code while merging. This contained the dialogBox. 

chunk

- I am restoring the above chunk. The below is the result 👇. I didn't notice any other glitch. If you find any, please inform.

keep

File: src/App.jsx [hunk 1]
```diff
@@ -360,6 +356,12 @@ useEffect(()=>{
             ))}
           </div>
         )}
+        {!loadingPosts &&
+          (user ? (
+          <ImgUpload user={user} snackBar={openSnackBar} />
```
Comment (narayan954) [inline src/App.jsx]: yes remove this and let line 240-253 of original stay , it seems like another bug

**Review by narayan954 — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (warmachine028) [reply src/App.jsx]: I already made the changes you requested. Is it as expected? or some more changes are required?

**Comment by narayan954** [comment]
Bro , with these changes , even after uploading is complete modal stays open on screen, state change issue

**Comment by warmachine028** [comment]
okay I will start fresh

**Pull request closed without merging**